network/discovery.go

```diff
@@ -177,6 +177,21 @@ func (d *discovery) isBootNode(id peer.ID) bool {
 
 	return false
 }
+
+func (d *discovery) addPeersToTable(nodes []string) error {
```

Comment:
Why should we error out even if 1 peer connection is unsuccessful?

We could be trying to connect to 10 peers, 1 of them will fail - we are supposed to stop trying to connect to the remaining 10?

This doesn't make sense to me, it's not a game breaking event

cc @Kourin1996

Reply:
Modified it, commit-[link](https://github.com/0xPolygon/polygon-edge/pull/343/commits/12c1341ba166c39ce334c47b8943697cfabdfccd)